appservice/src/utils/kuduUtils.ts

```diff
@@ -43,13 +43,13 @@ export async function ignore404Error(context: IActionContext, callback: () => Pr
 }
 
 // typings of queryParamsOptions allows booleans, but the ServiceClient only parses strings
-export function convertQueryParamsValuesToString(queryParams: { [key: string]: unknown }): { [key: string]: string } {
+export function convertQueryParamsValuesToString(queryParams: { [key: string]: string | boolean | number | object | undefined | null | bigint }): { [key: string]: string } {
     const keys: string[] = Object.keys(queryParams);
 
     for (const key of keys) {
         if (typeof queryParams[key] === 'object') {
             // skip objects, the ServiceClient will handle them
-        } else {
+        } else if (queryParams[key] !== undefined && queryParams[key] !== null) {
             queryParams[key] = String(queryParams[key]);
```

Comment:
I doubt it matters too much, but the behavior changes a bit here. If the value was undefined/null, we'd still define `queryParams[key]` as "undefined". Not sure if completely skipping those keys is going to cause any issue.